Dockerfile

```diff
@@ -24,4 +25,5 @@ RUN apt-get update && \
       rm -rf /var/lib/apt/lists/*
 WORKDIR /app
 COPY --from=build /project/engine .
+COPY --from=build /project/gen-genesis .
```

Comment:
Why gen-genesis is copied to the image?

Reply:
Like this the cli can directly execute it from the docker container.
The cli will be able to generate new genesis by using the script.

Reply:
cli calling internal scripts inside docker container grr....

Again this is a workaround for the poorly configuration we have right now. Probably, you don't want to pass and manage all those env in cli, so you have decided to copy the internal script as a "hack", instead of fix the pbm which is hard config management.

If that is the case, sorry I won't approve this PR :P

Reply:
The issue of the cli is not the envs or config but the code itself to generate the genesis transaction, keybase, validator identities. I don’t know any way to do it in JS for now. So the solution of this PR is to be able to call this script from the engine image. We could also compile the script and include it in the CLI but then it brings other issues